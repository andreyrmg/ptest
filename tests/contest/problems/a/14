500
9 100242
22 CJ3
34 Q51
35 WCK
35 SJ7
33 IFU
5 3010320
24 48IF
26 29M4
21 28ID
28 5GB
26 2HC5
6 325350
8 131445
25 3JN2
15 12516
3 2210120201
29 2HJB
16 A21
9 26145
26 3D57
27 23CC
18 8FC6
17 B6EB
8 71422
25 2HH2
13 8CB6
12 368
32 BPF
16 65F2
27 2J8J
30 23EN
27 8D6
3 1002
30 21LT
27 36AQ
4 13111223
23 1AI9
10 56923
11 156AA
10 58273
24 3I46
11 36424
23 1K2G
5 1211
35 1451
29 AFQ
17 161
34 1LT4
18 DB7
36 OIR
29 1588
17 1264
34 2TS
23 102I
32 1K77
24 2K2D
9 48254
33 11C3
30 10FR
5 1231234
6 1010524
4 31303132
5 1314331
4 33100212
14 A47A
13 8131
18 5A
22 2EDH
24 4HH6
17 DGF
30 1958
10 56474
29 1DO4
28 1FMN
5 2001414
30 145B
22 4C57
14 174D0
18 5GF3
13 13129
12 1A351
9 53743
16 7DDE
23 53K9
18 9HAF
25 BGO
8 5373
13 1C80C
34 9BU
17 55A2
30 1JKK
3 120101002
12 1B103
2 1100010000
2 10010100001
9 100443
29 1796
14 8D38
22 283B
18 48B9
10 32954
29 6OM
21 KD4
22 569L
11 19060
14 126C6
9 35602
17 8GB8
3 112102122
29 1FEN
36 1DTJ
27 29CA
32 1V1
14 17675
36 7HK
25 2K50
12 2A3A7
20 1EJF
22 1IH0
31 4I5
33 14DA
34 702
36 HGM
33 1A52
19 3897
10 45354
6 1050515
27 33F3
4 33111203
14 19B9D
3 2000221212
7 145653
18 6A9A
32 ITM
35 IHM
11 32294
5 140244
10 54060
28 2813
7 151214
32 1163
35 14N1
29 51K
10 34738
29 1JL5
29 HJR
11 6215
18 4F62
32 1JDV
3 1120220122
10 61859
32 14P6
31 23Q1
6 553150
20 539
21 51I7
22 11B7
31 256Q
18 2GD4
13 9157
5 123122
7 242125
15 13822
3 2000211111
16 E990
17 38FF
35 YT3
10 48593
8 125331
22 2KF0
23 6IJ
28 2LHP
30 S5K
33 15R4
3 2102011201
14 C959
31 1NK3
23 156C
17 23G2
14 78B4
8 151674
25 CA4
26 2MM6
36 24S
3 2010211022
18 A6B7
3 2021011012
7 6646
36 4KK
29 277N
36 8YK
36 4EK
28 L9K
15 63CA
15 8357
8 114
7 324652
11 18323
11 34347
24 86L
17 835
30 P24
18 GDE
21 42GF
12 18B1A
28 5JN
7 321104
31 TQ1
29 1G2K
10 22039
26 2HG
5 1133410
20 6J2I
9 78310
29 K36
24 2JC0
8 75562
16 9F6A
25 25E3
3 2101120
13 12A4A
22 3IH2
15 BD21
10 12378
13 1489B
29 NLD
9 44475
10 31102
23 4B21
8 133731
8 3703
27 374M
33 1FEV
2 1011011001011111
31 1PA5
35 SQO
13 A4B8
22 20C
35 YG4
24 3LL8
36 15FS
3 1122222000
9 87880
20 2JBF
11 6A36
36 A5H
2 1011001110001000
7 216513
17 3ACB
34 1CHM
6 414343
24 1ECL
29 2BOJ
5 2421203
26 F5P
30 1CDD
7 115625
23 5372
5 2301432
27 2JG3
27 28IA
2 110001001101001
29 11LB
36 98
35 MRB
4 13023123
13 7603
31 1E0U
15 D35E
35 168A
29 1B9B
25 2FD7
30 28JH
31 GJ6
16 ACD7
5 3123321
34 J7E
23 27A5
12 2B775
11 3A11A
12 28AA0
23 438H
17 477B
7 361334
32 LU0
9 42047
32 H08
17 2A8A
17 C53F
25 3C12
12 30423
6 50501
14 16779
14 C472
25 LAK
18 22H2
20 167E
34 12O2
36 XPD
17 84E8
23 1DLM
2 1111010000000011
10 63237
21 5B23
25 15FG
2 1111100111000101
11 26329
2 1010001111010111
18 1BG1
10 32255
28 1KFC
34 1I72
31 R9R
31 1ORT
19 3F93
17 F4F
15 1622
20 7D41
14 6AB4
29 1S6L
4 22101021
17 8BFF
34 WUA
19 2088
22 2474
16 56AB
32 R97
21 2015
34 15IX
36 6V6
34 N5I
32 1ELI
16 409E
25 36H7
22 55JI
28 22I4
20 4743
21 1AB0
17 C163
6 230334
26 10H3
8 23076
28 5C4
3 2200010021
28 2DQG
31 1K8G
2 1110111011000
25 H06
32 193R
27 35FH
12 15049
7 13646
8 11707
8 153316
14 5AD6
20 7I90
2 1101010010010010
27 N83
33 8AO
32 1EP0
19 6CHD
26 3269
25 2BI1
21 67I5
5 102144
5 122401
34 KT7
16 8C90
18 1CE4
28 190I
14 141AC
9 22322
10 4929
13 192C9
34 J80
10 10134
13 23A05
25 279M
33 1C0P
7 340061
30 22CG
4 21320331
25 FME
23 4HHB
5 1433
19 7EDG
6 1151532
23 163M
34 VSA
12 A7
9 52177
32 1GHL
21 ID8
4 30002221
7 210610
5 1312420
5 2144204
27 2G3D
14 2D79
29 1K05
17 6D87
32 1ALV
24 3A6I
35 3T9
13 22825
12 56B9
23 53CI
23 56IB
10 34834
10 30390
32 1RD1
2 1100010111100000
31 1DK6
23 1DM4
19 37FD
25 2399
34 15DG
12 13933
14 6
24 2BKE
19 254F
31 230T
11 1749
15 5BCA
3 2200222101
21 6K96
20 4A4E
24 162I
13 17508
9 30815
33 1GHR
20 72B1
23 283A
15 940E
4 32110222
30 6QM
34 99E
23 3EE8
26 4EF
3 2201022100
20 1D16
11 20783
16 3D60
16 5B41
5 1004333
27 2LHO
17 95E7
19 318
20 431B
26 18AC
17 56E1
5 2312304
22 4FF4
22 102I
11 37743
17 13F1
4 23211203
8 20323
35 E5X
4 33101022
6 404155
21 5GD
13 21117
19 FBA
6 402402
8 112007
2 1011101001100001
24 2EMG
15 142DA
8 123653
26 1HE0
21 5JHD
34 11RS
10 57734
14 11B24
2 1011110110101001
16 9846
28 2EK5
14 162B8
23 3AC4
3 1222202201
35 1G0W
17 CG2G
6 1024425
6 22330
6 415415
30 23CR
29 ONM
34 O11
34 1B0M
34 1FUH
27 2AG1
36 JAN
22 4163
29 1GG9
22 605G
17 2B98
29 2IDB
17 2971
