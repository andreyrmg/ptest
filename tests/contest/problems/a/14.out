14 6
3 1002
8 114
18 5A
12 A7
5 1211
5 1433
36 98
17 161
12 368
2 1100010000
22 20C
19 318
2 10010100001
3 2101120
26 2HG
8 3703
32 1V1
20 539
11 1749
17 835
7 6646
21 5GD
16 A21
36 24S
8 5373
26 4EF
6 22330
34 2TS
5 102144
23 6IJ
7 13646
17 DGF
29 51K
28 5C4
28 5GB
31 4I5
18 DB7
17 F4F
28 5JN
35 3T9
5 122401
15 1622
24 86L
5 123122
10 4929
8 11707
18 GDE
17 1264
19 FBA
5 140244
36 4EK
29 6OM
36 4KK
17 13F1
27 8D6
30 6QM
22 CJ3
6 50501
25 BGO
2 1110111011000
25 CA4
34 702
14 2D79
21 ID8
11 6215
8 20323
29 AFQ
36 6V6
33 8AO
21 KD4
11 6A36
12 56B9
18 1BG1
36 7HK
8 23076
25 FME
18 1CE4
10 10134
26 F5P
3 112102122
20 167E
25 H06
22 102I
34 99E
34 9BU
17 23G2
3 120101002
22 11B7
36 8YK
32 BPF
23 102I
10 12378
17 2971
18 22H2
17 2A8A
36 A5H
17 2B98
20 1D16
25 LAK
19 2088
21 1AB0
20 1EJF
9 22322
29 HJR
23 156C
23 163M
19 254F
16 3D60
14 5AD6
31 GJ6
5 1004333
13 7603
16 409E
28 L9K
29 K36
27 N83
18 2GD4
17 38FF
24 162I
35 E5X
32 H08
9 26145
13 8131
17 3ACB
23 1AI9
26 10H3
21 2015
14 6AB4
25 15FG
32 ITM
15 5BCA
23 1DLM
23 1DM4
6 230334
22 1IH0
13 8CB6
29 NLD
13 9157
33 IFU
9 30815
29 ONM
14 78B4
5 1133410
15 63CA
7 115625
17 477B
10 22039
11 156AA
16 56AB
24 1ECL
34 J7E
34 J80
21 28ID
32 LU0
30 P24
36 HGM
35 IHM
13 A4B8
23 1K2G
26 18AC
16 5B41
22 2474
19 37FD
19 3897
9 35602
20 2JBF
5 1231234
34 KT7
14 8D38
36 JAN
2 110001001101001
22 283B
30 S5K
11 18323
29 11LB
5 1312420
16 65F2
18 48B9
19 3F93
17 55A2
5 1314331
31 R9R
17 56E1
11 19060
34 N5I
6 325350
12 13933
30 10FR
9 42047
15 8357
34 O11
35 MRB
32 R97
23 27A5
18 4F62
14 A47A
22 2EDH
7 145653
23 283A
31 TQ1
29 1588
28 190I
7 151214
4 13023123
26 1HE0
12 15049
8 71422
9 44475
4 13111223
11 20783
34 Q51
10 30390
29 1796
30 145B
10 31102
3 1120220122
15 940E
22 2KF0
5 2001414
8 75562
6 402402
36 OIR
6 404155
16 7DDE
10 32255
9 48254
3 1122222000
10 32954
20 431B
25 2399
17 6D87
6 414343
6 415415
14 C472
29 1B9B
32 1163
28 1FMN
9 52177
24 2BKE
18 5GF3
13 12A4A
25 25E3
10 34738
14 C959
10 34834
20 4743
35 SJ7
35 SQO
30 1958
13 13129
9 53743
25 279M
16 8C90
29 1DO4
20 4A4E
12 18B1A
24 2EMG
7 210610
34 VSA
33 11C3
5 2144204
29 1FEN
11 26329
32 14P6
8 112007
29 1G2K
34 WUA
28 1KFC
30 1CDD
21 42GF
29 1GG9
7 216513
18 6A9A
12 1A351
25 2BI1
13 1489B
24 2JC0
16 9846
3 1222202201
24 2K2D
35 WCK
12 1B103
3 2000211111
3 2000221212
15 BD21
4 21320331
17 84E8
33 14DA
16 9F6A
5 2301432
25 2FD7
29 1JL5
22 3IH2
29 1K05
34 11RS
5 2312304
26 29M4
27 23CC
2 1010001111010111
4 22101021
23 3AC4
32 193R
3 2010211022
35 YG4
33 15R4
25 2HH2
34 12O2
35 YT3
17 8BFF
31 1DK6
8 123653
22 4163
31 1E0U
14 11B24
36 XPD
32 1ALV
8 125331
25 2K50
7 242125
17 8GB8
23 3EE8
16 ACD7
3 2021011012
15 D35E
30 1JKK
13 17508
5 2421203
14 126C6
10 45354
34 15DG
27 28IA
34 15IX
19 6CHD
8 131445
17 95E7
2 1011001110001000
28 22I4
6 553150
27 29CA
2 1011011001011111
11 32294
26 2HC5
33 1A52
8 133731
27 2AG1
21 51I7
24 3A6I
4 23211203
3 2102011201
5 3010320
2 1011101001100001
32 1ELI
32 1EP0
35 1451
29 1S6L
6 1010524
22 4C57
2 1011110110101001
35 14N1
10 48593
13 192C9
33 1C0P
31 1K8G
4 30002221
11 34347
32 1GHL
14 141AC
22 4FF4
28 2813
6 1024425
23 438H
35 168A
26 2MM6
2 1100010111100000
25 36H7
27 2G3D
21 5B23
5 3123321
18 8FC6
24 3I46
9 78310
34 1B0M
11 36424
31 1NK3
3 2200010021
32 1JDV
33 1FEV
3 2200222101
6 1050515
19 7EDG
3 2201022100
27 2J8J
32 1K77
27 2JG3
36 15FS
31 1ORT
34 1CHM
33 1GHR
10 54060
24 3LL8
31 1PA5
11 37743
8 151674
26 3269
25 3C12
2 1101010010010010
23 4B21
28 2DQG
29 277N
8 153316
21 5JHD
3 2210120201
27 2LHO
14 162B8
28 2EK5
30 21LT
7 321104
20 6J2I
17 B6EB
22 55JI
30 22CG
13 1C80C
14 16779
22 569L
10 56474
4 31303132
12 28AA0
10 56923
7 324652
20 72B1
30 23CR
30 23EN
11 3A11A
34 1FUH
10 57734
23 4HHB
18 9HAF
10 58273
9 87880
15 12516
14 174D0
4 32110222
29 2BOJ
14 17675
21 67I5
9 100242
12 2A3A7
25 3JN2
17 C163
9 100443
13 21117
16 E990
7 340061
24 48IF
34 1I72
17 C53F
18 A6B7
32 1RD1
28 2LHP
20 7D41
6 1151532
12 2B775
27 33F3
26 3D57
30 28JH
10 61859
2 1111010000000011
31 230T
4 33100212
35 1G0W
4 33101022
15 13822
23 5372
23 53CI
12 30423
4 33111203
23 53K9
13 22825
27 35FH
10 63237
31 23Q1
20 7I90
17 CG2G
29 2HJB
27 36AQ
2 1111100111000101
22 605G
27 374M
29 2IDB
23 56IB
36 1DTJ
34 1LT4
21 6K96
31 256Q
15 142DA
7 361334
14 19B9D
13 23A05
24 4HH6
